theforeman.org/pipelines/infra/updateJobs.groovy

```diff
@@ -14,7 +14,7 @@ pipeline {
                 ])
 
                 virtEnv('jjb-venv', 'pip install pbr')
```

Comment:
Is this still needed? I imagine pip should be able to do this.

Reply:
you would, right? (it didn't last time we tried)